assets/src/styles/campaigns/themes/_theme_forest.scss

```diff
@@ -519,4 +519,49 @@ body.theme-forest {
   .video-block {
     @include campaign_video($forest-dark-green);
   }
+
+  .page-template table.spreadsheet-table {
+    color: #fff;
+
+    tr {
+      background: #A0D654;
+      color: #1A1A1A;
+
+      &:nth-child(odd) {
+        background: #B6ED5F;
+        color: #1A1A1A;
+      }
+
+      &:hover {
+        background-color: #074365;
+        color: white;
+      }
+
+      &:first-child {
+        background: #2CAF4E;
+        color: #FFFFFF;
+      }
+    }
+
+    .spreadsheet-table-search {
+      border-color: #407ea2;
+    }
+  }
+
+  .spreadsheet-default-sort {
+    @include forest-button($white, $forest-dark-green);
```

Comment:
Just wondering, is there a specific reason this is not used on all themes? arctic and oceans are not using is.

Reply:
Removed these overrides in favor of using `btn-secondary` along with a single override for this class.